### ISSUE TITLE
Support for custom delimiters

### ISSUE THREAD

**Issue by amansx** [ef2k/raymond#1]
Not sure if it would be a simple regex configuration or a more complicated approach is warranted, but would be a nice feature to have

**Comment by ef2k** [comment]
Yeah, definitely. I'm not sure how complicated it would be, but let's leave this one open.
